public/app/plugins/panel/logs/types.ts

```diff
@@ -52,6 +54,23 @@ export function isOnNewLogsReceivedType(callback: unknown): callback is onNewLog
   return typeof callback === 'function';
 }
 
+export function isOnLogOptionsChange(callback: unknown): callback is onLogOptionsChangeType {
+  return typeof callback === 'function';
```

Comment:
isn't this just a fancy type assertion?

Reply:
Not entirely sure about the comment, but for these callbacks we've been only making sure we're passing callable references.